esp/services/ws_topology/ws_topologyService.cpp

```diff
@@ -236,6 +236,16 @@ bool CWsTopologyEx::onTpLogFile(IEspContext &context,IEspTpLogFileRequest  &req,
     return true;
 }
 
+void CWsTopologyEx::checkFilePath(const char *file, const char *fileType, const char *compType, const char *compName)
```

Comment:
minor, really just personal preference, and feel free to disagree, I would name the functions "validate" rather than "check" when their primary purpose is security related.

Reply:
I will change it.